sql/ballots/ballots.go

```diff
@@ -148,3 +148,28 @@ func CountByPubkeyLayer(db sql.Executor, lid types.LayerID, pubkey []byte) (int,
 	}
 	return rows, nil
 }
+
+// GetRefBallot gets a ref ballot for a layer and a pubkey.
+func GetRefBallot(db sql.Executor, epochID types.EpochID, pubkey []byte) (ballotID types.BallotID, err error) {
+	firstLayer := epochID.FirstLayer()
+	lastLayer := firstLayer.Add(types.GetLayersPerEpoch())
+	rows, err := db.Exec(`
+		select * from ballots 
+		where layer between ?1 and ?2 and pubkey = ?3
```

Comment:
`between` keyword inclusive or exclusive? would be nice to have a test for this case as well